test/e2e/tests/test_db_instance.py

```diff
@@ -110,6 +116,22 @@ def test_crud_postgres13_t3_micro(
             assert len(aws_res['DBInstances']) == 1
             dbi_rec = aws_res['DBInstances'][0]
 
+        time.sleep(CHECK_STATUS_WAIT_SECONDS)
+
+        # Before we update the DBInstance CR below, let's check to see that the
+        # DbInstanceStatus field in the CR has been updated to something other
+        # than 'creating', which is what is set after the initial creation.
+        # The CR's `Status.DBInstanceStatus` should be updated because the CR
+        # is requeued on successful reconciliation loops and subsequent
+        # reconciliation loops call ReadOne and should update the CR's Status
+        # with the latest observed information.
+        # https://github.com/aws-controllers-k8s/community/issues/923
+        cr = k8s.get_resource(ref)
+        assert cr is not None
+        assert 'status' in cr
+        assert 'dbInstanceStatus' in cr['status']
+        assert cr['status']['dbInstanceStatus'] != 'creating'
```

Comment:
Not a big deal, but the status should normally become `available` in here
```suggestion
        assert cr['status']['dbInstanceStatus'] == 'available'
```

Reply:
@A-Hilaly :) originally I had it like that, but unfortunately, after `creating`, RDS sets a DBInstance to a variety of other statuses *before* `available`, including statuses like `backing-up` :)